Lib/test/test_importlib/test_windows.py

```diff
@@ -111,8 +112,10 @@ def test_module_not_found(self):
 class WindowsExtensionSuffixTests:
     def test_tagged_suffix(self):
         suffixes = self.machinery.EXTENSION_SUFFIXES
-        expected_tag = ".cp{0.major}{0.minor}-{1}.pyd".format(sys.version_info,
-            re.sub('[^a-zA-Z0-9]', '_', get_platform()))
+        abi_flags = "t" if sysconfig.get_config_var("Py_NOGIL") else ""
```

Comment:
Now I'm confused. On Windows, `sys.abiflags` doesn't exist? It doesn't look practical that `sys.abiflags` doesn't exist to look for PYD files :-(

Maybe we should now add this flag?

Reply:
The awkward bit is that debug libraries are indicated as a `_d` prefix like, `foo_d.cp313t-win_amd64.pyd` instead of as part of the abiflags. Still might be useful.

Reply:
We can maybe change that in Python 3.13 to make it consistent.